compare-images

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+import json
+import optparse
+import os
+import sys
+from tabulate import tabulate
+
+from kubedifflib import (
+  get_differing_images,
```

Comment:
What was the motivation for putting `get_differing_images` in the pkg and not in this file?